doc/developer/build_release.rst

```diff
@@ -82,14 +82,21 @@ Checkout the code:
     git checkout master
     git pull origin master
 
-Edit the ``doc/integrator/update_application.rst`` file to change the default version
-for the ``version.cfg``.
+Edit the ``doc/integrator/update_application.rst`` file to change the default version.
+
+Update the version of c2cgeoportal in the
+``c2cgeoportal/scaffolds/update/CONST_requrements.txt`` and in the
```

Comment:
CONST_requrements.txt => CONST_requirements.txt
ditto on next line

Instead of "in the ccc.txt and the ddd.txt files to the <release>" I would rather write "to <release> in files ccc.txt and ddd.txt".